CMakeLists.txt

```diff
@@ -89,8 +70,6 @@ if (NOT BUILD_PYTHON_LIB) # Not taking the setup.py route, just building libs
 	endif ()
 endif()
 
-
-
 if(BUILD_PYTHON_LIB)
```

Comment:
Just use `find_package(pybind11 CONFIG REQUIRED)` since pybind11 will now get installed as a dependency instead of getting cloned in `lib/pybind11`